feat(curriculum): add football player card tsx project (#67076)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-build-a-football-player-card-builder/69e8b295388ebbd517692bb5.md

---
id: 69e8b295388ebbd517692bb5
title: Step 6
challengeType: 0
dashedName: step-6
---

# --description--

> Add another `div` inside the `div` element with the class name `card-header`. This one won't have any class name. Inside that `div`, add another `div` with the class name `card-rating` and the text `98`.

Reviewer: ```suggestion
Add a `div` inside the `.card-header` element. Inside it, add another `div` with the `className` of `card-rating` and the text `98`.
```

# --hints--

> You should add a `div` with no class name inside `.card-header`.

Reviewer: ```suggestion
You should add a `div` with no `classname` inside `.card-header`.
```

```js
assert.exists(document.querySelector('.card-header > div'));
```

> You should add a `div` with the class name `card-rating` inside the unnamed `div`.

Reviewer: ```suggestion
You should add a `div` with the `className` of `card-rating` inside the unnamed `div`.
```
